yarn.lock

```diff
@@ -5683,20 +5685,6 @@ http-proxy-middleware@0.19.1:
   version "0.19.1"
   resolved "https://registry.yarnpkg.com/http-proxy-middleware/-/http-proxy-middleware-0.19.1.tgz#183c7dc4aa1479150306498c210cdaf96080a43a"
   integrity sha512-yHYTgWMQO8VvwNS22eLLloAkvungsKdKTLO8AJlftYIKNfJr3GK3zK0ZCfzDDGUBttdGc8xFy1mCitvNKQtC3Q==
-  dependencies:
-    http-proxy "^1.17.0"
-    is-glob "^4.0.0"
-    lodash "^4.17.11"
-    micromatch "^3.1.10"
-
-http-proxy@^1.17.0:
```

Comment:
Why is that? I just tried `git add is-cidr` locally and it did not remove me anything from here.

Reply:
Sorry, why did you expect it to remove anything?

Reply:
I did not but it is being removed by this PR

Reply:
@mareklibra  also this dependency has been removed `follow-redirects@^1.0.0`. Do you want me to revert the `yarn.lock`? Please let me know if you want me to do something else.